tools/expotools/src/commands/PublishPackages.ts

```diff
@@ -0,0 +1,202 @@
+import { Command } from '@expo/commander';
```

Comment:
I agree, I've been thinking about doing the same structure of `expotools/src/commands` as it is in `expo-cli`: https://github.com/expo/expo-cli/tree/master/packages/expo-cli/src/commands
So that every command would have its own directory and then only commander's command would be defined in `index.ts` 😄 What I did here is kind of a step forward to do this as I moved all helpers/tasks/constants to different files. For now, moving the action to separate file would be cluttering too much - it would fit more into directory per-command structure which I'd like to do very soon.